src/itemset.js

```diff
@@ -234,7 +242,15 @@ define([
                 "SelectDynamic": util.extend(mugTypes.Select, {
                     typeName: 'Single Answer Lookup Table',
                     typeChangeError: function (mug, typeName) {
-                        return typeName === "MSelectDynamic" ? "" : "Can only change to a dynamic multiple answer";
+                        if (typeName.match(/^M?Select$/)) {
+                            if (mug.form.getChildren(mug).length > 0) {
+                                return "Cannot change to Multiple/Single Choice " +
+                                      "question if it has Choices. " +
+                                      "Please remove all Choices and try again.";
+                            }
+                            return '';
+                        }
+                        return typeName === "MSelectDynamic" ? "" : "Can only change to a dynamic single answer";
```

Comment:
this looks like an unintentional change: dynamic multiple answer -> dynamic single answer